README.md

```diff
@@ -214,7 +214,7 @@ ZF2 Services
 
 #### ZF\Rest\AbstractResourceListener
 
-This abstract class is the base implementation of a [Resource](zfrestresource) listener.  Since
+This abstract class is the base implementation of a [Resource](https://github.com/zfcampus/zf-rest/blob/master/src/AbstractResourceListener.php) listener.  Since
```

Comment:
Better would be to use `#zfrestresource`, which is what was intended (links to an anchor within the README itself). I'll make that change on merge.